compiler/src/dotty/tools/dotc/Driver.scala

```diff
@@ -64,19 +64,33 @@ class Driver {
 
   protected def sourcesRequired: Boolean = true
 
-  def setup(args: Array[String], rootCtx: Context): (List[AbstractFile], Context) = {
+  protected def command: CompilerCommand = ScalacCommand
+
+  /** Setup context with initialized settings from CLI arguments, then check if there are any settings that
+   *  would change the default behaviour of the compiler.
+   *
+   *  @return A tuple of Option of List and Context.
+   *  If there is no setting preventing us from continuing compilation, function returns Some(List)
```

Comment:
This a method, not a function:
```suggestion
   *  If there is no setting like `-help` preventing us from continuing compilation, this method returns Some(...)
```

Reply:
Will fix